app/models/article.rb

```diff
@@ -285,9 +285,9 @@ def comments_blob
   end
 
   def username
-    return organization.slug if organization
+    return organization.slug.downcase if organization
 
-    user.username
+    user.username.downcase
```

Comment:
Could we add a quick test for this? I think maybe [here](https://github.com/forem/forem/blob/master/spec/models/article_spec.rb#L522-L531) or [here](https://github.com/forem/forem/blob/master/spec/models/user_spec.rb#L440-L452)?